videointelligence/CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 
 [1]: https://pypi.org/project/google-cloud-videointelligence/#history
 
+## 1.12.0
+
+11-08-2019 09:32 PST
+
+
+### Implementation Changes
+- Make `features` a positional parameter in `annotate_video`, update retry config, make AnnotateVideo nonidempotent (via synth) ([#9440](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/9440))
+
+### New Features
+- Add celebrity recognition support (via synth) ([#9612](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/9612))
```

Comment:
```suggestion
- Add celebrity recognition support (via synth). ([#9612](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/9612))
```

Reply:
As with the Vision release PR, we need to add periods since the tooling which converts conventional commits isn't yet there.